src/modules/control/include/control/lat_controller.h

```diff
@@ -6,34 +6,34 @@
 #include "math.h"
 #include <vector>
 #include <iostream>
+#include <limits>
 
 
 //Namespace declaration
 namespace fusionad
 {
-  namespace control
-  {
-    namespace lat_controller
-    {
-      class stanley
-      {
-        public:
-          stanley();
-          ~stanley();
-        private:
-          double controlGain;
-          //TODO: Declare a custom ROS control message output after message definition is complete
-          //ROS --> steering message.
-          typedef Eigen::Matrix<double, 4, 2> pathMatrix42d; 
-          double computeHeadingError(const double &vehHeading, const double &pathHeading);
-          double computePathHeading(const Eigen::pathMatrix42d &trajectory, const int &targetIndex);
-          double computeNonLinControlTerm(const double &linear_velocity, const double &position, const double &waypoint);
-          double computeCrossTrackError(const double &routeTheta, const double &dx, const double &delta_y);
-          double computeSteeringAngle(const Eigen::Vector2d &vehPos, const Eigen::Vector2d &targetPos, 
-                                      const std::vector<double> &routeX, const std::vector<double> &routeY,
-                                      const double &vehSpeed, const int &wpIndex);
-      };
-    }
-  }
+namespace control
+{
+namespace lat_controller
+{
+class stanley
+{
+  public:
+    stanley();
+    ~stanley();
+  private:
+    //TODO: Declare a custom ROS control message output after message definition is complete
```

Comment:
Is this linked to another issue? Just don't want it to get lost in the woods as the code expands